src/Microsoft.ML.Data/Transforms/ValueMappingTransformer.cs

```diff
@@ -32,14 +32,63 @@
 
 namespace Microsoft.ML.Transforms.Conversions
 {
+    /// <summary>
+    /// The ValueMappingEstimator is a 1-1 mapping from a key to value. This particular class load the mappings from an <see cref="IDataView"/>.
```

Comment:
>key [](start = 58, length = 3)

reference to KeyType

Reply:
Here term `Key` is used as general term as used in dictionary/map. Its not referencing ML.Net KeyType specifically here. Let me know if you meant something else?

---
In reply to: [251594412](https://github.com/dotnet/machinelearning/pull/2232#discussion_r251594412) [](ancestors = 251594412)